frontend/src/app/app.module.ts

```diff
@@ -1,62 +1,62 @@
+import { HomepageOfferComponent } from 'app/components/homepage-offer/homepage-offer.component';
+import { LoginComponent } from 'app/components/login/login.component';
+import { MessagesComponent } from 'app/components/messages/messages.component';
+import { CreateOfferComponent } from 'app/components/offers/create-offer/create-offer.component';
+import { OfferDetailComponent } from 'app/components/offers/offer-detail/offer-detail.component';
+import { OrganizationComponent } from 'app/components/organization/organization.component';
+import { OrganizationContactComponent } from 'app/components/organization/organization-contact/organization-contact.component';
+import { OrganizationDetailsComponent } from 'app/components/organization/organization-details/organization-details.component';
+import { OrganizationOffersListComponent } from 'app/components/organization/organization-offers-list/organization-offers-list.component';
+import { OrganizationsComponent } from 'app/components/organizations/organizations.component';
+import { OrganizationCreateComponent } from 'app/components/organization/organization-create/organization-create.component';
+import { OrganizationsListComponent } from 'app/components/organizations/organizations-list/organizations-list.component';
+import { PasswordResetComponent } from 'app/components/password-reset/password-reset.component';
+import { PasswordResetConfirmComponent } from 'app/components/password-reset/password-reset-confirm.component';
+import { RedirectComponent } from 'app/components/redirect/redirect.component';
+import { RegisterComponent } from 'app/components/register/register.component';
+import { AboutUsComponent } from 'app/components/static/about-us.component';
+import { FaqOrganizationsComponent } from 'app/components/static/faq-organizations.component';
+import { FaqVolunteersComponent } from 'app/components/static/faq-volunteers.component';
+import { OfficeComponent } from 'app/components/static/office/office.component';
+import { RegulationsComponent } from 'app/components/static/regulations.component';
+import { UserProfileComponent } from 'app/components/user-profile/user-profile.component';
+import { LoggedInGuard } from 'app/guards/loggedInGuard.service';
+import { LoggedOutGuard } from 'app/guards/loggedOutGuard.service';
+import { HttpWithCredentialsInterceptor, HttpXsrfInterceptor } from 'app/http-interceptor';
+import { ContactResolver } from 'app/resolvers';
+import { AuthService } from 'app/services/auth.service';
+import { ContactService } from 'app/services/contact.service';
+import { MessagesService } from 'app/services/messages.service';
+import { MetatagsService } from 'app/services/metatags.service';
+import { OffersService } from 'app/services/offers.service';
+import { OrganizationService } from 'app/services/organization.service';
+import { UserService } from 'app/services/user.service';
+import { WindowFactory, WindowService } from 'app/services/window.service';
```

Comment:
What do you think about the idea to define paths in compilerOptions in tsconfig.json to remove `app/` from all those imports?

Reply:
That would cause problem with `environments/environment` imports.

Reply:
I was sure that paths allow to keep baseUrl for imports like 'environments/environment' but also to have different imports paths for different parts of an app, but maybe I misunderstood smth.